addons/bestja_account_deletion/models.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+from openerp import models, fields
+import uuid
+
+
+class Volunteer(models.Model):
+    _inherit = 'res.users'
+    STATES = [
+        ('active', 'aktywny'),
+        ('not_activated', 'nieaktywowany'),
+        ('deleted', 'usuniety')
+    ]
+
+    active_state = fields.Selection(STATES, default='not_activated', store=True, string="Stan")
```

Comment:
I guess we can use the `selection_add` argument here (see commit 2a94eb5b9f74a0004e2 for usage example).